### DISCUSSION TITLE
DietPi-Software | yt-dlp

### DISCUSSION THREAD

**Discussion by rgabbo** [MichaIng/DietPi#5670] [Software Request]
#### Formal software information
- Software title | yt-dlp
- Short description | download video from youtube and other sites
- GitHub URL (if available) | https://github.com/yt-dlp/yt-dlp

#### Are there similar/alternative software titles available with DietPi-Software?

- youtube-dl

#### What makes your requested software better than the above solutions, if available?

- youtube-dl is not actively mantained and the main maintainer left the project (see https://github.com/ytdl-org/youtube-dl/issues/29965). [Yt-dlp](https://github.com/yt-dlp/yt-dlp) is at the moment the most active fork. I suggest to consider switching to this software.

**Comment by MichaIng** [comment]
Many thanks for reporting. Indeed it seems like a good direct replacement for youtube-dl.

**Reply by Ast3risk-ops** [reply]
It should be pretty easy to implement, as installation is VERY simple

**Comment by MichaIng** [comment]
Another request from the forum: REMOVED INVALID LINK

**Reply by Joulinar** [reply]
some more request's 

https://dietpi.com/forum/t/replace-youtube-dl-with-yt-dlp/11727

**Reply by Ast3risk-ops** [reply]
nice

**Comment by MichaIng** [comment]
PR up: #6380

**Reply by MichaIng** [reply]
Merged!
